synapse/lib/cell.py

```diff
@@ -611,15 +611,15 @@ async def getHealthCheck(self):
     async def getDmonSessions(self):
         return await self.cell.getDmonSessions()
 
-    @adminapi()
+    @adminapi(log=True)
```

Comment:
why do we want to log a list call that doesn't change anything?

Reply:
removed this